### DISCUSSION TITLE
macOS installation succeeds, but running the binary fails with 'segmentation fault'.

### DISCUSSION THREAD

**Discussion by GiantRavens** [openai/whisper#903] [General]
Perhaps I missed a configuration step? Installation via pip succeeds as do all dependencies, only running the binary always throws a segmentation fault error on Ventura. I'm on an Intel MacBook Pro - perhaps the ARM version was installed?

**Comment by GiantRavens** [comment]
Resolved with swizzling conda environment, and installing openai and whisper via pip3 which pulled in the correct dependencies.

**Comment by glangford** [comment]
Does it seg fault regardless of the input file?

One Mac issue I know of is this, fyi
https://developer.apple.com/forums/thread/719949
